ResearchKit/Common/ORKAnswerFormat.m

```diff
@@ -1756,20 +1756,21 @@ - (NSNumber *)minimumNumber {
 - (NSNumber *)maximumNumber {
     return @(self.textChoices.count);
 }
-- (NSNumber *)defaultNumber {
+- (id<NSObject, NSCopying, NSCoding>)defaultAnswer {
     if (_defaultIndex < 0 || _defaultIndex >= _textChoices.count) {
         return nil;
     }
-    return @(_defaultIndex);
+   
+    return @[[self.textChoices objectAtIndex:_defaultIndex].value];
```

Comment:
Use `_textChoices`; and use `_textChoices[_defaultIndex]` modern syntax instead of `[_textChoices  objectAtIndex: _defaultIndex]`?